ga4gh/backend.py

```diff
@@ -37,8 +38,8 @@ def _parsePageToken(pageToken, numValues):
     return values
 
 
-def _getVariantSet(request, variantSetIdMap):
-    variantSetId = request.variantSetId
+def _getVariantSet(requestOrCompoundId, variantSetIdMap):
```

Comment:
Yeah, whatever choice we make we're in an intermediate state where something can be overloaded.

'compoundId' can refer to either an:id:that:looks:like:this or a instance that derives from the new type CompoundId

A refactor/cleanup will be needed after these routes are finished.